docs/containers/kubernetes-env-vars.md

```diff
@@ -0,0 +1,96 @@
+---
+Area: containers
+ContentId: 8c1ee589-25bb-4c2d-ab8d-7ccfa7b4e282
+PageTitle: Use Kubernetes service environment variables for service to service communication
+DateApproved: 02/12/2021
+MetaDescription: Learn how to use Kubernetes service environment variables with Bridge to Kubernetes to enable service-to-service communication as a non-elevated user in a Kubernetes cluster
+---
+# Use Kubernetes service environment variables
+
+When you communicate with another service in the same Kubernetes cluster, for example with an HTTP request, you typically use the service name in the URL for the request. Bridge to Kubernetes reroutes traffic by modifying the host name resolution to redirect network traffic to its own version of the services. This works in most scenarios, but fails in the case where the Bridge to Kubernetes process has restricted privilege, such as when the request originates from a non-elevated user account or when using VS Code Remote SSH. This is because in order to enable the name resolution for redirected services, Bridge to Kubernetes needs to modify the hosts file, but that is not possible when Bridge to Kubernetes runs from a non-elevated user account. To work around this issue, you can write your code to use the Kubernetes service environment variables instead of a hardcoded service name.
```

Comment:
Is there a way to break this up into two paragraphs?
Also there are several "This" starting sentences. Acrolinx will recommend you clarify what "This" is. 
For example "This redirection works...:  and "The redirection failure is because...."

Reply:
One idea would be to split the paragraph with an H2 that calls out the value proposition of using service env vars.
Something like "Avoid redirection failures"